requirements.txt

```diff
@@ -1,7 +1,14 @@
+alembic
+beautifulsoup4
```

Comment:
@pypingou how can I check dependencies of other packages automatically?  Is there any commands or should I have to look into their <code> setup.py </code> file individually.